storage-proofs/core/src/api_version.rs

```diff
@@ -0,0 +1,29 @@
+use std::str::FromStr;
+
+use anyhow::{self, Result};
+use semver::Version;
+
+#[derive(Debug, Copy, Clone, Eq, PartialEq)]
+pub enum ApiVersion {
```

Comment:
lets add a manual implementation for `Debug` and `Display` that prints them as their semver version aka `1.1.0`

Reply:
Ok, in that case, I think we should also more strongly consider full versioning (e.g. `ApiVersion::V1_0_0` instead of `ApiVersion::V1_0`)

Reply:
fair enough